src/src/FHSS.h

```diff
@@ -35,42 +35,50 @@ extern int32_t FreqCorrection;
 #define FreqCorrectionMax 200000
 #define FreqCorrectionMin -200000
 
-void ICACHE_RAM_ATTR FHSSsetCurrIndex(uint8_t value);
+
 
 uint8_t ICACHE_RAM_ATTR FHSSgetCurrIndex();
 
+#ifndef SX127x_FREQ_STEP
+#define SX127x_FREQ_STEP 61.03515625
```

Comment:
does not belong here. Radio specific stuff, imo.

Reply:
Ok, left in radio lib